scripts/provisioning/s3_setup_constants.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+"""Constants"""
+LDAP_USER = "cn={},dc=seagate,dc=com"
+LDAP_URL = "ldapi:///"
+
+CONFSTORE_OPENLDAP_CONST_KEY = 'openldap'
```

Comment:
can these come from config file

Reply:
this is constants file.. you mean should we make this file a config file ?

Or some other already existing file ?

Reply:
I have changed this constant file to a yaml config file.